### DISCUSSION TITLE
v. 2 silent install [SOLVED]

### DISCUSSION THREAD

**Discussion by skylas173** [brackets-cont/brackets#144] [Q&A] [answered]
Hello, is it possible to install the new version 2 silently?

**Comment by abose** [comment, marked as answer]
* For Mac, it should be possible with some scripting
* For Windows, ~~it is not possible currently~~ .  in windows as @charlypa mentioned, you could do it by executing the installer by adding /qn to silent install for example Brackets.2.0.1ml.exe /qn. You may need to launch your script with administrative privileges

Can you explain more on the usecase?

**Reply by skylas173** [reply]
Thank you.
I would like to allow users on volatile VDI machines to install it via script.

**Reply by charlypa** [reply]
@skylas173 
in `windows`   you could do it by executing the installer by adding `/qn` to silent install for example  `Brackets.2.0.1ml.exe /qn`.   You may need to launch your script with administrative privileges

**Reply by skylas173** [reply]
Thank you! works perfectly
